Sofa/Component/Collision/Detection/Intersection/src/sofa/component/collision/detection/intersection/LocalMinDistance.cpp

```diff
@@ -186,27 +186,34 @@ int LocalMinDistance::computeIntersection(Line& e1, Line& e2, OutputVector* cont
     double alpha = 0.5;
     double beta = 0.5;
 
+    // If line are not parallel
     if (det < -1.0e-30 || det > 1.0e-30)
     {
+        // compute the parametric coordinates along the line
         alpha = (b[0]*A[1][1] - b[1]*A[0][1])/det;
         beta  = (b[1]*A[0][0] - b[0]*A[1][0])/det;
+
+        // if parameters are outside of ]0,1[ then there is no intersection
+        // the intersection is outside of the edge supporting point.
         if (alpha < 1e-15 || alpha > (1.0-1e-15) ||
             beta  < 1e-15  || beta  > (1.0-1e-15) )
             return 0;
     }
-    else
```

Comment:
You removed the `else`, which is not without effect. I just want to make sure it is on purpose, and has a reason.

Reply:
Indeed, I think this is a mistake. It will always set alpha and beta to 0.5.

Reply:
ouch :)